include/vapor/ImageParams.h

```diff
@@ -95,14 +102,44 @@ class PARAMS_API ImageParams : public RenderParams
     SetValueLong( _orientationTag, "set orientation value", val );
   }
 
+  //! Get the TMS level of detail
+  //! \retval int Currently selected TMS level of detail.  Value of 0 means we are computing a default value.
```

Comment:
The default value returned is -1. What does this mean?

Reply:
Sorry - my comment was obsolete.  -1 means we're computing a default value.  Comment has been updated.